packages/wonder-blocks-icon/components/icon.js

```diff
@@ -84,7 +84,7 @@ export default class Icon extends React.PureComponent<Props> {
         // double-quotes on commit. So the aria-label prop isn't included in
         // props validation.
         // eslint-disable-next-line react/prop-types
-        const {"aria-label": ariaLabel} = this.props;
+        const {"aria-hidden": ariaHidden, "aria-label": ariaLabel} = this.props;
```

Comment:
@kevinbarabash As you mentioned in the previous PR, I'm going to modify this logic as part of WB-580 to make these props more generic.